code/modules/vending/_vending.dm

```diff
@@ -759,33 +746,200 @@ GLOBAL_LIST_EMPTY(vending_products)
   * * I - the item being loaded
   * * user - the user doing the loading
   */
-/obj/machinery/vending/proc/canLoadItem(obj/item/I,mob/user)
-	return FALSE
-/**
-  * Is the passed in user allowed to load this vending machines compartments
-  *
-  * Arguments:
-  * * user - mob that is doing the loading of the vending machine
-  */
-/obj/machinery/vending/proc/compartmentLoadAccessCheck(mob/user)
```

Comment:
Where did this code go

Reply:
chef thing no longer used

Reply:
honestly should have kept it and overwritten this proc for custom...